src/test/javascript/spec/component/programming-exercise/programming-exercise-update.component.spec.ts

```diff
@@ -215,6 +219,29 @@ describe('ProgrammingExerciseUpdateComponent', () => {
             expect(programmingExerciseService.automaticSetup).toHaveBeenCalledWith(entity);
             expect(entity.title).toBe('My Exercise');
         }));
+
+        it('should fail on error', async () => {
+            // GIVEN
+            const entity = new ProgrammingExercise(undefined, undefined);
+            entity.id = 1;
+            jest.spyOn(programmingExerciseService, 'update').mockReturnValue(
+                throwError(
+                    new HttpResponse({
+                        headers: new HttpHeaders({ 'X-artemisApp-alert': 'error-message' }),
+                    }),
+                ),
+            );
+            const alertSpy = jest.spyOn(alertService, 'addAlert');
+            comp.programmingExercise = entity;
+            comp.backupExercise = {} as ProgrammingExercise;
+            comp.programmingExercise.course = course;
+            // WHEN
+            comp.save();
+
+            // THEN
+            expect(comp.isSaving).toBeFalsy();
```

Comment:
```suggestion
            expect(comp.isSaving).toBeFalse();
```
The dev guidelines recommend that the specific assertion should be used since `Falsy` also includes `undefined`.
Can this attribute be `undefined` here? Other tests below seem to use the recommended assertion, too.